app/controllers/election.ts

```diff
@@ -63,10 +64,16 @@ export const create = (req, res) =>
     .then((election) => {
       const alternatives = req.body.alternatives;
       if (alternatives && alternatives.length) {
-        return Bluebird.map(alternatives, (alternative) => {
-          alternative.election = election;
-          return Alternative.create(alternative);
-        }).then((createdAlternatives) => {
+        const promises = [] as Array<unknown>;
+        
+        for (const alternative of alternatives) {
+          promises.push(async () => {
+            alternative.election = election;
+          return Alternative.create(alternative);  
+          });
+        }
+
+        Promise.all(promises).then((createdAlternatives) => {
```

Comment:
I'd prefer a more functional approach, and get rid of the `for` loop and `push` stuff.

Something like this:

```suggestion
        Promise.all(
        	alternative.map(a => {
        		a.election = election;
        		return Alternative.create(alternative);
        	}
        ).then((createdAlternatives) => {
```

Also, should not type it as `unknown`. If you don't want to deal with the types now, I'd rather you have it as `any` so it gives an error and you can fix it later.